EORTC/load_stage.sql

```diff
@@ -0,0 +1,1146 @@
+                                        invalid_reason)
+
+SELECT DISTINCT concept_code_1,
+                concept_code_2,
+                vocabulary_id_1,
+                vocabulary_id_2,
+                relationship_id,
+                valid_start_date,
+                valid_end_date,
+                invalid_reason
+FROM (SELECT DISTINCT cs2.concept_name                AS concept_name_1,
+                      cs2.vocabulary_id               AS vocabulary_id_1,
+                      cs2.concept_code                AS concept_code_1,
+                      'Direction of'                    AS relationship_id,
+                      NULL                            AS invalid_reason,
+                      CURRENT_DATE                    AS valid_start_date,
+                      TO_DATE('20991231', 'yyyymmdd') AS valid_end_date,
+
+
+                      cs.concept_name                 AS concept_name_2,
+                      cs.vocabulary_id                AS vocabulary_id_2,
+                      cs.concept_code                 AS concept_code_2
+      FROM concept_stage cs
+               JOIN concept_relationship_stage crs
+                    ON cs.concept_code = crs.concept_code_1
+                        AND cs.vocabulary_id = crs.vocabulary_id_1
+                        AND cs.concept_class_id = 'Question'
+                        AND cs.standard_concept is null
+               JOIN concept_stage csi
+                    ON csi.concept_code = crs.concept_code_2
+                        AND csi.vocabulary_id = crs.vocabulary_id_2
+                        AND csi.concept_class_id = 'Question'
+              AND csi.standard_concept ='C'
+               JOIN concept_relationship_stage crsi
+                    ON crsi.concept_code_2 = csi.concept_code
+                        AND csi.vocabulary_id = crsi.vocabulary_id_1
+
+               JOIN concept_stage cs2
+                    ON crsi.concept_code_1 = cs2.concept_code
+                        AND cs2.vocabulary_id = crsi.vocabulary_id_2
+                        AND cs2.concept_class_id = 'Direction'
+      ) AS q_to_issue
+;
+
+--Stages should be OK
+SELECT *
+FROM qa_tests.check_stage_tables();
+--Stages are OK
+
+
+-- Working with concept_manual table
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.ProcessManualConcepts();
+    END
+$_$;
+
+--Working with replacement mappings
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.CheckReplacementMappings();
+    END
+$_$;
+
+--Add mapping from deprecated to fresh concepts
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.AddFreshMAPSTO();
+    END
+$_$;
+
+--Deprecate 'Maps to' mappings to deprecated AND upgraded concepts
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.DeprecateWrongMAPSTO();
+    END
+$_$;
+
+--Add mapping from deprecated to fresh concepts for 'Maps to value'
+DO
+$_$
+    BEGIN
+        PERFORM VOCABULARY_PACK.AddFreshMapsToValue();
+    END
+$_$;
+
```

Comment:
Add our standard 'at the end all tables ...'

Reply:
Added